packages/flutter_app/macos/Runner/Info.plist

```diff
@@ -30,5 +30,8 @@
 	<string>MainMenu</string>
 	<key>NSPrincipalClass</key>
 	<string>NSApplication</string>
+	<key>MDItemKeywords</key>
+	<string>fab, flutter_app_template,ふらったー,てんぷれーと</string>
+</dict>
 </dict>
```

Comment:
```suggestion
	<string>fab, flutter_app_template,ふらったー,てんぷれーと</string>
</dict>
```

Since there seems to be one too many `</dict>`, does it look like I should remove it?
Any reason to knowingly include `</dict>`?